DNN Platform/Library/Entities/Users/UserRegistrationEmailNotifier.cs

```diff
@@ -35,7 +38,9 @@ public static void NotifyUser(UserInfo user)
             switch (PortalSettings.Current.UserRegistration)
             {
                 case (int)PortalRegistrationType.PrivateRegistration:
-                    NotifyUser(user, MessageType.UserRegistrationPrivate);
+                    NotifyUser(user, CurrentUser.Value != null && CurrentUser.Value.IsSuperUser ?
```

Comment:
IMO the superuser check is not sufficient, users might be added by a site admin OR superuser

Reply:
Thanks @sleupold.  I commented above.

Flow is:
Check permissions -> Create User -> If User Created successfully - > Notify User by Email
Notification is a last step in the process, means we already checked all permissions.
Here we only need to define if it was self-registration or not.

Reply:
AFAIR there was a specific treatment of superusers before, to prevent passwords being sent to them. This is not necessary anymore, as we do not send passwords and the check might be removed.